gossip/keys.go

```diff
@@ -45,6 +45,9 @@ const (
 	// KeyConfigPermission is the permission configuration map.
 	KeyConfigPermission = "permissions"
 
+	// KeyConfigUser is the user configuration map.
+	KeyConfigUser = "user"
```

Comment:
yeah, we seem to be pretty inconsistent when it comes to those. There are at least three constants that often differ:
- key prefix (from keys/constants.go)
- gossip key (from gossip/keys.go)
- admin endpoint (from server/admin.go)

I'm happy changing them to be the same in all three places, as well as consistent amongst themselves (eg: abbreviated or not, singular or plural)